docs/particles/particle_class.rst

```diff
@@ -74,6 +74,8 @@ These properties are often returned as a |Quantity| in SI units.
 <Quantity 3.888e+08 s>
 >>> iron56.binding_energy.to('GeV')
```

Comment:
@namurphy this should really be called `nuclear_binding_energy` to distinguish from the sum of the ionization energies for this particle!!  I will open an issue...